sdk/src/main/java/com/ifountain/opsgenie/client/model/beans/Contact.java

```diff
@@ -58,20 +67,20 @@ public Contact withTo(String to) {
         return this;
     }
 
-    public Contact withEnabled(Boolean enabled) {
-        this.enabled = enabled;
+    public Contact withEnabled(Status status) {
+        this.status = status;
         return this;
     }
 
-    public Contact withDisabledReason(String disabledReason) {
-        this.disabledReason = disabledReason;
+    public Contact withId(String id) {
+        this.id = id;
         return this;
     }
 
     public enum Method {
         EMAIL(OpsGenieClientConstants.API.EMAIL), SMS(OpsGenieClientConstants.API.SMS),
-        VOICE(OpsGenieClientConstants.API.VOICE),
-        MOBILE_APP(OpsGenieClientConstants.API.MOBILE_APP);
+        VOICE(OpsGenieClientConstants.API.VOICE);
+       // MOBILE_APP(OpsGenieClientConstants.API.MOBILE_APP);
```

Comment:
is this not supported now?

Reply:
Yes, I have removed this as per mentioned in document.
Details mentioned for this API:
This parameter is the contact method of user and should be one of email, sms or voice. Please note that adding mobile is not supported from API.